### DISCUSSION TITLE
Edge error: Cannot lookup route

### DISCUSSION THREAD

**Discussion by fpolli** [adonisjs/core#4418] [Help]
Hi, I am getting the above error when I try to use an edge template with the route() helper in it.

          Error: Cannot lookup route "moms.join"
              at anonymous (D:\www\seeds\resources\views\moms\pages\home.edge:95:0)

This is the line in the Edge template:
<a class='button' href="{{ route('moms.join') }}">Join!</a>

That route shows up in list:routes:
GET    /join (moms.join) .................................................. #controllers/join_controller.join

What am I doing wrong? Please let me know what additional information is needed.

Thank you.

EDIT:
I should probably add that the 'moms' at the start of that route name is for a domain:

router
  .group(() => {
    router.get('/join', [JoinController, 'join']).as('join')
  })
  .domain('moms.test')
  .as('moms')

**Comment by craigharman** [comment]
You need to add the domain to the route. eg. `route('moms.join', [], { domain: 'moms.test' })` if you are using the edge helper or 
```ts
router
  .builderForDomain('moms.test')
  .make('moms.join')
 ```
